informus/rules/patients.md

```diff
@@ -1,65 +1,117 @@
+
+This also holds true for shorter trips in and out of the unit. For example, let's imagine a patient has been on T03 for multiple days in a specific bed, before being moved 12 hours ago to another location in the hospital (could be another ward, or the operating theatre, imaging etc.), before being transferred back to the same bed on T03 10 hours ago. In this scenario we would use all data generated by the patient on T03 in the last 24 hours (so 0-10 hours ago, as well as 12-24 hours ago), but ignore any data generated during the 2 hours they were off the unit.
+
+Equally, we must ensure that any data 'gaps' in the last 24 hours (due to the patient being outside the unit, in another location in the hospital), don't negatively impact the scores on each tile. There is no data for these patients on T03 in these gaps, so they shouldn't contribute to the overall score.
+
+Tiles should still show data if there are no patients currently on the unit (as long as there have been some moved / discharged patients present in the last 24 hours). Clicking on a tile should show an empty floorplan in this scenario.
+
+If there are no current patients and there haven't been any in the last 24 hours, then tiles should display text that says 'no data available'.
+
+The time window is always exactly 24 hours (even over a daylight savings transition).
+
+### Floorplan (i.e. `latest` data)
+
+The floorplan (i.e. the view with beds as individual coloured rectangles), should only include data for patients currently on the unit (i.e. patients that have since been discharged or moved to another location in the hospital should be excluded). 
+
+As above, we still need to ensure that we only use data generated within the current unit, excluding any that may have been generated while the patient was in another location in the hospital. Bear in mind that current patients may have previously been moved in/out of the unit, as well as within the unit within our time period of interest. 
+
+### Metric charts (i.e. `24hr` / `72hr` data)
+
+The metric charts (shown when clicking on a bed in the floorplan), should only include data for patients currently on the unit (i.e. patients that have since been discharged or moved to another location in the hospital should be excluded). 
+
+As above, we still need to ensure that we only use data generated within the current unit, excluding any that may have been generated while the patient was in another location in the hospital. Times when the patient was off the unit will appear as 'missing' data in the chart.
```

Comment:
I think there is potential for this to change as we learn more about how the backfilling works for each individual metric. But for now I think this logic is broadly correct.

Reply:
Yes I agree - I imagine there's more detail to be added to all these sections as we get deeper into the actual code implementation